pages/1.13/security/oss/login/index.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: layout.pug
+navigationTitle: Login
+title: Login
+excerpt: Logging in to your DC/OS cluster
+menuWeight: 20
+
+---
+
+<!-- The source repository for this topic is https://github.com/dcos/dcos-docs-site -->
+
+# User login
+
+With DC/OS, login is the process of exchanging user credentials for a [DC/OS Authentication token](/1.13/security/oss/authentication/authentication-token/).
+
+Obtaining a DC/OS Authentication is required by any user before being able to use a DC/OS cluster. In Open DC/OS the life-time of an Authentication token is limited to five days. Once the Authentication token expires the user must log in again.
+
+DC/OS handles multiple user types. Users can be configured via the [IAM API](/1.13/security/oss/iam-api/); see [User Management](/1.13/security/oss/user-management/).
+
+Different login methods exist for different user types but each one yields a DC/OS authentication token:
+
+* **External user login**: An external user logs in by supplying an `OpenID Connect Token` that has been issued by Mesosphere's Auth0 instance after verification of the user credentials by an external login provider (Google, GitHub or Microsoft).
+* **Local user login**: A local user logs in by supplying a `password` which is compared to the `password hash` stored inside DC/OS.
+* **Service login**: A services logs in by supplying a `service login token` of which the signature is verified using the service account `public key` stored inside DC/OS.
```

Comment:
A service

Reply:
Remove ticks around "service login token", instead say for example "A service logs in by supplying a short-lived so-called "service login token" ...

Reply:
And again this listing kind of suggests that a service account is not a local user account :/

Reply:
See above comment